API/users/UserController.js

```diff
@@ -97,7 +97,7 @@ exports.create = async function (req, res) {
                         res.status(500).json(err);
                     } else {
                           res.clearCookie('_oAuthId');
-                          res.cookie('_oAuthId', user.oAuthId, {expire: (Date.now+1000*60*60*24)}).json({
+                          res.cookie('_oAuthId', user.oAuthId, {expires: (Date.now+1000*60*60*24)}).json({
```

Comment:
Are you sure it is "expires"? According to [https://www.geeksforgeeks.org/http-cookies-in-node-js/] it is "expire".